material-design-catalog.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Material Design Catalog
+extra_css:
+  - /css/catalog.css
+sidebar: home_sidebar
+permalink: /material-design-catalog/
+---
+
```

Comment:
I'd love to add a disclaimer up at the top, something like "(Flutter implements a set of material design widgets, but you don't need to use material design. Flutter's base widgets provide extensible starting points to build customer UIs with Flutter. To learn more, check out ______.)"